components/downgrade_modal/index.scss

```diff
@@ -0,0 +1,26 @@
+#DowngradeModal {
+    padding-top: 25px;
+}
+
+.background-svg {
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    width: 100%;
+    height: 100%;
+}
+
+.ProcessPayment-progress {
+    width: 387px;
+    height: 4px;
+    margin: 0 auto;
+    margin-top: 60px;
+    background: rgba(34, 64, 109, 0.12);
```

Comment:
Nit: You could use the colour variables that are defined in _css_variables.scss here. Something like `rgba(var(--secondary-blue-rgb), 0.12)`

Reply:
Yeah, good idea. Less "magic" numbers, more pre-defined values. Thanks! :)